src/components/Home.js

```diff
@@ -1,11 +1,65 @@
-import React from 'react';
-
-const Home = ({ handleTokenCreation }) => (
-  <main>
-    <button onClick={handleTokenCreation} className="button-home">
-      Create a Shopping List
-    </button>
-  </main>
-);
+import React, { useState } from 'react';
+import { db } from '../lib/firebase';
+
+const Home = (props) => {
+  const [value, setValue] = useState('');
+  const [message, setMessage] = useState('');
+  const setToken = props.setToken;
```

Comment:
Why are you guys creating a variable to store the setToken props? Can't you guys call it from there like `              props.setToken(userToken)`?

Reply:
Also, everywhere else in the codebase, we destructure props right away. Please, could you do it in this file, too?